src/lib/entities/audio/model/specs/audio-instance.spec.ts

```diff
@@ -41,4 +41,14 @@ describe('Audio playback API', () => {
     expect(get(currentTime)).toEqual(get(duration));
     pause();
   });
+
+  it('plays twice without pause', async () => {
+    playDefault();
+    await new Promise((r) => setTimeout(r, 1000));
```

Comment:
Someone pointed in a previous PR that it's better to use `jest.advanceTimersByTime`, and that would require
```ts
jest.useFakeTimers();
jest.spyOn(global, 'setTimeout');
```
somewhere on the top. See [`$lib/shared/ui/specs/snackbar.spec.ts`](https://github.com/aabounegm/cast/blob/5cbc1fd9bf61dcdf5ca0b28d74875a6cf965ae80/src/lib/shared/ui/specs/snackbar.spec.ts) for example usage.